WorldEdit/Commands/Copy.cs

```diff
@@ -11,7 +12,15 @@ public Copy(int x, int y, int x2, int y2, TSPlayer plr)
 
 		public override void Execute()
 		{
-			string clipboardPath = Tools.GetClipboardPath(plr.User.Name);
+			string clipboardPath = Tools.GetClipboardPath(plr.User.ID);
+
+			if (!Tools.NewClipboardStruct(clipboardPath))
+			{
+				if (File.Exists(clipboardPath))
```

Comment:
SaveWorldSection is used to save undo files and it supports only new struct. Copy supports both old and new structs